leaflet-geoman.d.ts

```diff
@@ -703,6 +742,8 @@ declare module 'leaflet' {
             allowSelfIntersection?: boolean;
         }
 
+        type VertexValidationHandler = (e: { layer: L.Layer, marker: L.Marker, event: any }) => boolean;
```

Comment:
is this correct? It should be a function with the arguments `layer`, `marker`, `event` and returns a boolean value.